src/log.c

```diff
@@ -886,6 +886,7 @@ LogData *log_data_link_block(ConfigItem_link *link)
 	json_object_set_new(j, "hostname", json_string_unreal(link->outgoing.hostname));
 	json_object_set_new(j, "ip", json_string_unreal(link->connect_ip));
 	json_object_set_new(j, "port", json_integer(link->outgoing.port));
+	json_object_set_new(j, "class", json_string_unreal(link->class->name));
```

Comment:
I had considered adding a null check in for link->class, but from reading conf.c it doesn't seem like should ever be NULL (as it should get replaced by the default class), but feel free to adjust if that's a bad assumption.